frontend/src/components/PDFViewer/index.ts

```diff
@@ -1 +1 @@
-export { default } from './PDFViewerContainer';
+export { default as PDFViewer } from './PDFViewerContainer';
```

Comment:
Try to remove curly brackets, it worked for me in case of `export { default } from './PDFViewerContainer';`

<img width="636" alt="image" src="https://user-images.githubusercontent.com/44285339/164077576-9e2b9d32-7793-4d2d-9799-62094fdd5e30.png">